tests/MongoDB.Driver.Tests/SslSettingsTests.cs

```diff
@@ -61,9 +62,11 @@ public void TestCheckCertificateRevocation()
             Assert.Throws<InvalidOperationException>(() => { settings.CheckCertificateRevocation = checkCertificateRevocation; });
         }
 
-        [Fact]
+        [SkippableFact]
         public void TestClientCertificates()
         {
+            RequirePlatform.Check().SkipWhen(SupportedOperatingSystem.MacOS, SupportedTargetFramework.NetStandard15);
```

Comment:
Mac doesn't contain some required libraries from `X509Certificate2`. So skipped it for now